src/python/platforms/android/flash.py

```diff
@@ -52,6 +56,31 @@
 FLASH_REBOOT_WAIT = 5 * 60
 
 
+def download_latest_build(build_info, image_regexes):
+  """Download the latest build artifact for the given branch and target."""
+  # Check if our local build matches the latest build. If not, we will
+  # download it.
+  build_id = build_info['bid']
+  target = build_info['target']
+  image_directory = environment.get_value('IMAGES_DIR')
```

Comment:
read this env var once from the caller, and have it passed as an argument rather than reading it again here.

Reply:
Done